libs/fluid-elements/switch/src/lib/switch.ts

```diff
@@ -46,7 +62,11 @@ export class FluidSwitch extends LitElement {
   @property({ type: Boolean, reflect: true })
   disabled = false;
 
-  /** Whether the switch is active or not */
+  /** The value attribute of the native input element */
+  @property({ type: String, reflect: false })
```

Comment:
Value can be reflected.
```suggestion
  @property({ type: String, reflect: true })
```